src/Extensions/Logging/Internal/OmexLogsEventSource.cs

```diff
@@ -0,0 +1,198 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+using System;
+using System.Diagnostics;
+using System.Diagnostics.Tracing;
+using System.Globalization;
+using System.Text;
+using Microsoft.Extensions.Logging;
+using Microsoft.Omex.Extensions.Abstractions;
+
+namespace Microsoft.Omex.Extensions.Logging
+{
+	[EventSource(Name = "Microsoft-OMEX-Logs")]
+	internal sealed class OmexLogsEventSource : EventSource
+	{
+		static OmexLogsEventSource()
```

Comment:
>static [](start = 2, length = 6)

Note that static constructors have strange behaviours in an inheritance hierarchy and code analysis typically warns against using them. A better pattern would be initialise the variables within the non-static constructor the first time it is called (assuming you are doing it in a thread-safe manner) or to use lazy initialisation on these properties.

Reply:
I would like to avoid multiple executions of this code, the class is sealed so don't think there would be inheritance problems